dispatch/apps/events/models.py

```diff
@@ -68,15 +70,33 @@ def save_image_from_url(self, url):
 
 
 @receiver(post_update, sender=Event)
+def send_approved_email(sender, instance, **kwargs):
+
+    if not instance.is_approved:
```

Comment:
`is_approved` takes care of this, as it is never set to false. I'll rename it to `is_approved_email`